test/client/js/helpers/ircmessageparser/findLinks.js

```diff
@@ -8,12 +8,12 @@ const {
 
 describe("findLinks", () => {
 	it("should find url", () => {
-		const input = "irc://freenode.net/thelounge";
+		const input = "irc://irc.example.com/thelounge";
 		const expected = [
 			{
 				start: 0,
 				end: 28,
```

Comment:
This should be 31 now (length of the link).

Reply:
Oh, I forgot to commit this. Already fixed.